tests/postgres/worker.test.ts

```diff
@@ -1,11 +1,11 @@
 import { PluginEvent } from '@posthog/plugin-scaffold/src/types'
```

Comment:
I'm slightly confused with the naming of the folders here. Some (or all?) tests inside `postgres` are ones that to the best of my understanding operate on things not explicitly related to postgres... Things like checking if the VMs, workers, etc work well. They should be in a higher ingestion-agnostic folder IMO to emphasize this.

Reply:
Also, there are broken imports in the `plugin`, `queue` and `vm` tests. Why not move them back?

Reply:
Problem is, these tests will require a bit of a redesign to work with both pipelines. So for now separate.